client/src/components/Overview.jsx

```diff
@@ -27,34 +27,34 @@ function generateStars(average) {
   const starBar = [];
   for (let i = 0; i < filledStars; i++) {
     starBar.push(<BsStarFill />);
-  } 
-  
+  }
+
```

Comment:
appreciate the readability changes, but avoid working on other peoples components, it will cause merge conflicts when they pull from the main branch